docs/gettingstarted/verifier_callback.rst

```diff
@@ -2,15 +2,15 @@
 Getting Verifier
 ================
 
-As part of OAuth to allow OOB (Out of band) applications to have access to the account
-we have a link that they will click, follow the instructions and then copy the verifier
-into the application which we then relay to the server with other tokens. The server
-will them provide us with the credentials that we can use.
+For OAuth to allow OOB (Out of band) applications to have access to an account,
+we must provide a link, instructions, and a means of copying the verifier into an application
+and relaying it to the server with other tokens. The server
```

Comment:
yeah, I agree. I was trying to go for "the means of copying... and relaying" from the list of things one must "provide" (links, instructions, and means). I think its grammatically correct albeit confusing. And, anyway, too many ands for my taste.  Maybe it bares repetition? Something like, 

+we must provide a link, instructions, a means of copying the verifier into an application,
+and **a means** of relaying it to the server with other tokens.

**OR!** I'm thinking splitting it into what the user needs and what the application/server needs:

+First, we must provide a link and instructions for the user. Then we must provide a means of copying the verifier into an application and relaying it to the server with other tokens.

Or is this all too silly/irrelevant anyway?